portfolio-master-responsive/code/style.css

```diff
@@ -0,0 +1,458 @@
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Rubik', sans-serif;
+  margin: 0 auto;
+  max-width: 1100px;
+}
+
+.container {
+  max-width: 1100px;
+  margin: 0 auto;
+}
+
+p {
+  font-family: 'Rubik', sans-serif;
+  font-size: 16px;
+  text-align: center;
+  margin: 24px -10px;
+  color: #444448;
+}
+
+/*** HEADER ***/
+
+header {
+  height: 80px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 16px;
+}
+
+header p {
+  z-index: 2;
+  text-transform: uppercase;
+  letter-spacing: 3px;
+  color: #444448;
+  padding: 0;
+  text-align: left;
+}
+
+.hamburger-container {
+  position: relative;
+  top: 0px;
+  width: 30px;
+  height: 30px;
+  background-color: blue;
+}
+
+.hamburger-container:hover ~ .hamburger-nav {
+  position: absolute;
+  width: 100px;
+  height: 300px;
+  background-color: yellow;
+}
+
+.hamburger {
+  position: absolute;
+  top: 0;
+  right: 0px;
+  display: flex;
+  width: 30px;
+  height: 30px;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.hamburger div {
+  width: 30px;
+  height: 5px;
+  background: #444448;
+}
+
+.hamburger-nav {
+  display: none;
+  flex-direction: column;
+  background-color: #444448;
+}
+
+.hamburgernav-container {
+  display: none;
+  background-color: yellowgreen;
+  width: 100vw;
+  height: 300px;
+}
+
+.hamburger-nav a {
+  z-index: 1;
+  color: white;
+  padding: 16px;
+}
+
+.screen {
+  width: 180px;
+  display: none;
+  margin-left: 80px;
+}
+
+.mobile {
+  width: 42px;
+  padding-right: -5px;
+}
+
+h1 {
+  text-transform: uppercase;
+  letter-spacing: 3px;
+  text-align: center;
+  color: #444448;
+  padding: 0 16px;
+  font-size: 16px;
+  font-weight: normal;
+  padding-top: 16px;
+}
+
+nav {
+  display: none; 
+}
+
+nav a {
+  color: #444448;
+  text-decoration: none;
+  margin-left: 16px;
+}
+
+nav a:hover {
+  background-color: yellow;
+}
+
+/*** RESPONSIVE NAV ***/
+
+
+
+
+
+/*** SECTION GALLERY ***/
+
+.gallery .container {
+  background-color: #444448;
+  display: flex;
+  margin-right: 30px;
+  flex-flow : wrap;
```

Comment:
This CSS property takes two arguments (flex-direction and flex-wrap). If you only want to specify flex-wrap, you can use that:
`flex-wrap: wrap;`